data/tutorials/gs_01_a_first_hour_with_ocaml.md

```diff
@@ -70,32 +82,91 @@ $ ocaml
 ```
 
 Note that `#use` is not part of the OCaml language proper; it's an instruction
-to the OCaml toplevel only.
+to the OCaml toplevel only. Also, for `ocaml` or `utop` to find your file, you
+may need to start `ocaml` or `utop` inside the directory of your file, use a
+relative path that takes into account the directory you started the toplevel in,
+or use an absolute path.
+
+### Using the OCaml Playground
```

Comment:
Would it make sense to move this section in a separate page, before the "Get up and running" ? So people can try it and see if they like it before installing anything? This should probably be advertised better on the Learn page (currently only reachable from the top side menu)

Reply:
Having a tutorial running in the playground would be great indeed!